### PR TITLE
Add flake8-bugbear in checks

### PR DESCRIPTION
**Pull request by justinchuby**
### Description

Add flake8-bugbear in checks

### Motivation and Context

It checks for bug prone coding patterns.

### CODE REVIEW

**Comment by justinchuby** [comment]
@jcwchen

**Comment by justinchuby** [comment]
All checks passed

**Review by jcwchen — APPROVED** [review]
LGTM. Thank you for introducing flake8-bugbear and fixing existing errors!

**Comment by justinchuby** [comment]
Do we need another reviewer for this?

**Review by gramalingam — APPROVED** [review]
Thanks!

File: docs/Operators.md [hunk 1]
```diff
@@ -19168,8 +19168,6 @@ data = np.array(
     dtype=np.float32,
 )
 reduced = np.maximum.reduce(data, axis=axes, keepdims=keepdims == 1)
-# print(reduced)
```
Comment (gramalingam) [inline docs/Operators.md:19171]: Not sure about this ... may be this was intentional, to include the expected output in the documentation?

Comment (jcwchen) [reply docs/Operators.md:19171]: > to include the expected output in the documentation

Yes that is the most likely reason... If it is really useful, we can still keep it on demand.

**Pull request merged**